src/Ref.ts

```diff
@@ -99,6 +100,23 @@ export function getRefAt(document: vscode.TextDocument, position: vscode.Positio
     }
   }
 
+
+  regex = NoteWorkspace.rxMarkdownHyperlink();
+  range = document.getWordRangeAtPosition(position, regex);
+  if (range) {
+      ref = document.getText(range);
```

Comment:
Can you add some comments that explain what these regex replacements are doing? And/or factor into a well named function? Perhaps add some test cases for the behavior?